sdk/program/src/sysvar/mod.rs

```diff
@@ -26,6 +26,9 @@ pub fn is_sysvar_id(id: &Pubkey) -> bool {
         || instructions::check_id(id)
 }
 
+/// Sysvars that can be obtained via a syscall using `.get()`
+pub static SYSCALL_SUPPORTED_SYSVARS: &[Pubkey] = &[clock::ID, rent::ID];
```

Comment:
Any reason to not include all sysvars?

Reply:
In order to make these sysvars available via a syscall the runtime has to load their corresponding accounts for all transactions because it does not know whether they will be needed or not.  I figured initially we can limit the sysvars to the ones we know we need and are most used by folks, and expand as needed as the account caching stuff gets better.

Reply:
The latest changes also only support Clock and Rent, but it is easy to add the rest.  If the direction looks good I can add additional sysvars if folks feel like they should be